### ISSUE TITLE
[BUG] Internet connection interface

### ISSUE THREAD

**Issue by sharma-shray** [eythaann/Seelen-UI#378]
**Describe the bug**

Win10, No wifi connection is displayed, only “no wifi networks found” when hovering over the internet icon, I assume this is for laptops, I assume there must be a different interface on PCs?

**To Reproduce**

**Expected behavior**

**Screenshots**

**Comment by eythaann** [comment]
yeah, basically the same message is shown independent if you have or not a WLAN interface.

**Comment by datgausaigon** [comment]
Also happens with Windows 11.

<img width="733" height="273" alt="Image" src="https://github.com/user-attachments/assets/30ff8dea-5cc3-47aa-a254-e70b7f8eddd0" />

Thanks
🐻

**Comment by Legion2016** [comment]
Same

**Comment by hiran-filho** [comment]
same

i'm used last version 2.4.9
